src/fragments/lib/auth/flutter/getting_started/10_preReq.mdx

```diff
@@ -11,11 +11,15 @@
 
 <Block name = "Developer Preview (Mobile, Web & Desktop)">
 
-* A Flutter application targeting Flutter SDK >= 2.17.0 (stable version) with Amplify libraries integrated
-  * An iOS configuration targeting at least iOS 13.0
-  * An Android configuration targeting at least Android API level 24 (Android 5.0) or above
+* A Flutter application targeting Flutter SDK >= 3.0.0 with Amplify libraries integrated
+  
+  The following are also required, depending on which platforms you are targeting:
+
+  * An iOS configuration targeting at least iOS 13.0 and XCode version >=13.2
+  * An Android configuration targeting at least Android API level 24 (Android 7.0) or above
   * Any browser supported by Flutter web
-  * Any MacOS, Linux or Windows OS meeting Flutter minimums.
+  * Any macOS or Windows OS meeting Flutter minimums. 
+  * Any Ubuntu Linux distribution meeting Flutter minimums.
   * For a full example please follow the [project setup walkthrough](/lib/project-setup/create-application)
```

Comment:
I feel like this line becomes a bit confusing since we already have the callout at the top regarding project setup. Should we maybe remove it from stable and dev preview? what is this trying to say?

Reply:
This would apply for other sections where we use this language